src/components/modal/ModalPro.vue

```diff
@@ -7,27 +7,36 @@
 
 			<h2 class="card__meta"> Current space: {{ quota }}</h2>
 
-			<div v-if="!isPro" class="card__meta">
+                        <p v-if="!isPaid">By continuing you agree to our <a href="/terms.html" target="_blank" rel="noopener noreferrer">Terms of Service</a>.</p>
+                        <div v-if="!showCard" class="options_container">
+			    <div class="card__meta options">
 				<h3>Pro Account</h3>
 				<ul>
-					<li>50 GB of hyper secure storage</li>
-					<li>All our bundled private applications</li>
-					<li>5 GBP / month</li>
+				    <li>50 GB of hyper secure storage</li>
+				    <li>All our bundled private applications</li>
+				    <li>5£ / month</li>
 				</ul>
-
-
-			</div>
+                                <AppButton @click.native="updateCard(53687091200)" :disabled="isPro" type="primary" block accent>{{proButtonText}}</AppButton>
+			    </div>
+                            <div class="card__meta options">
+				<h3>Visionary Account</h3>
+				<ul>
+				    <li>500 GB of hyper secure storage</li>
+				    <li>All our bundled private applications</li>
+				    <li>25£ / month</li>
```

Comment:
use &#x00A3; for uk pound symbol

Reply:
i mean the numeric character reference
\# &#x00A3;

Reply:
done